tests/test_all.py

```diff
@@ -59,6 +61,19 @@ def test_os_release(auto_container):
         ("VERSION_ID", OS_VERSION),
         ("PRETTY_NAME", OS_PRETTY_NAME),
     ):
+        if OS_VERSION == "latest" and var_name == "VERSION_ID":
+            # on openSUSE Tumbleweed that is the an ever changing snapshot date
+            # just check whether it starts with current year
+            assert (
+                int(
+                    auto_container.connection.run_expect(
+                        [0], f". /etc/os-release && echo ${var_name}"
+                    ).stdout.strip()[:4]
+                )
+                == datetime.datetime.now().year
+            )
+            continue
```

Comment:
This makes it rather un-obvious that you are skipping the `pretty_name` check on Tumbleweed.

Reply:
I can put that back if there's value in that.

Reply:
I'm actually not checking the pretty_name check already, it is conditionalized on the version variable in the if above.